support image with alpha channels

Some of OCR-D's processors produce derived images with transparency (in masked parts like after rotation or polygonal cropping). Alas, the current implementation of this module's OCR-D processor cannot consume them:

```
  File "/local/ocr-d/ocrd_all/venv/lib/python3.10/site-packages/sbb_binarize/ocrd_cli.py", line 158, in cli
    return ocrd_cli_wrap_processor(SbbBinarizeProcessor, *args, **kwargs)
  File "/local/ocr-d/ocrd_all/venv/lib/python3.10/site-packages/ocrd/decorators/__init__.py", line 110, in ocrd_cli_wrap_processor
    run_processor(processorClass, ocrd_tool, mets, workspace=workspace, **kwargs)
  File "/local/ocr-d/ocrd_all/venv/lib/python3.10/site-packages/ocrd/processor/helpers.py", line 88, in run_processor
    processor.process()
  File "/local/ocr-d/ocrd_all/venv/lib/python3.10/site-packages/sbb_binarize/ocrd_cli.py", line 109, in process
    bin_image = cv2pil(self.binarizer.run(image=pil2cv(page_image), use_patches=True))
  File "/local/ocr-d/ocrd_all/venv/lib/python3.10/site-packages/sbb_binarize/ocrd_cli.py", line 35, in pil2cv
    return cv2.cvtColor(pil_as_np_array, color_conversion)
cv2.error: OpenCV(4.6.0) /io/opencv/modules/imgproc/src/color.simd_helpers.hpp:92: error: (-2:Unspecified error) in function 'cv::impl::{anonymous}::CvtHelper<VScn, VDcn, VDepth, sizePolicy>::CvtHelper(cv::InputArray, cv::OutputArray, int) [with VScn = cv::impl::{anonymous}::Set<3, 4>; VDcn = cv::impl::{anonymous}::Set<3, 4>; VDepth = cv::impl::{anonymous}::Set<0, 2, 5>; cv::impl::{anonymous}::SizePolicy sizePolicy = cv::impl::<unnamed>::NONE; cv::InputArray = const cv::_InputArray&; cv::OutputArray = const cv::_OutputArray&]'
> Invalid number of channels in input image:
>     'VScn::contains(scn)'
> where
>     'scn' is 2
```

IMO it should suffice reducing transparency to white before proceeding with OpenCV import.

--- a/sbb_binarize/ocrd_cli.py
+++ b/sbb_binarize/ocrd_cli.py
@@ -30,6 +30,10 @@ def cv2pil(img):
 
 def pil2cv(img):
     # from ocrd/workspace.py
+    if img.mode in ('LA', 'RGBA'):
+        newimg = Image.new(img.mode[:-1], img.size, 'white')
+        newimg.paste(img, mask=img.getchannel('A'))
+        img = newimg
     color_conversion = cv2.COLOR_GRAY2BGR if img.mode in ('1', 'L') else  cv2.COLOR_RGB2BGR
     pil_as_np_array = np.array(img).astype('uint8') if img.mode == '1' else np.array(img)
     return cv2.cvtColor(pil_as_np_array, color_conversion)

--- a/sbb_binarize/sbb_binarize.py
+++ b/sbb_binarize/sbb_binarize.py
@@ -35,6 +35,8 @@ class SbbBinarizer:
 
         self.model_files = glob('%s/*.h5' % self.model_dir)
         if not self.model_files:
+            self.model_files = glob('%s/*/' % self.model_dir)
+        if not self.model_files:
             raise ValueError(f"No models found in {self.model_dir}")
         
         self.models = []

